src/main/java/me/xmrvizzy/skyblocker/SkyblockerMod.java

```diff
@@ -73,6 +75,11 @@ public void onInitializeClient() {
         WikiLookup.init();
         ItemRegistry.init();
         NEURepo.init();
+        try {
```

Comment:
The try catch block should be moved into the actual file itself, and also shouldn't throw a runtime exception which'd cause a game crash but rather log an error and ideally disable the feature